modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -1569,11 +1569,27 @@ public void testDeleteObjectField() throws Exception {
 
 		_assertDeleteObjectField(true, systemObjectDefinition, "baker");
 
-		_objectDefinitionLocalService.deleteObjectDefinition(
-			customObjectDefinition);
 		_objectDefinitionLocalService.deleteObjectDefinition(
 			systemObjectDefinition);
 
+		// User cannot delete metadata object field
```

Comment:
The comments in a test method should be ordered alphabetically.

Reply:
I would rename the comment to: 
// Delete metadata object field

I think it is similar to the existing comments.